plugins/catalog-backend-module-aws/README.md

```diff
@@ -4,7 +4,56 @@ This is an extension module to the plugin-catalog-backend plugin, providing an
 `AwsOrganizationCloudAccountProcessor` that can be used to ingest cloud accounts
 as `Resource` kind entities.
 
-## Getting started
+The plugin also supports discovering catalog entities located in an S3 Bucket. See the [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for more details.
 
-See [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for details on how to install
-and configure the plugin.
+## For Ingesting AWS accounts
+
+1. Install this plugin:
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/backend @backstage/plugin-catalog-backend-module-aws
+```
+
+2. Import the plugin to `catalog.ts` and add the `AwsOrganizationCloudAccountProcessor`
+
+```TypeScript
+// packages/backend/src/plugins/catalog.ts
++ import {AwsOrganizationCloudAccountProcessor} from '@backstage/plugin-catalog-backend-module-aws';
```

Comment:
The formatting of the code seems not to follow the prettier formatting used for the project.

Reply:
hmm I ran `npx prettier --write README.md` locally and pushed these changes, and the Verify workflow logs are happy in the `prettier` [step](https://github.com/backstage/backstage/actions/runs/5918320281/job/16047533982?pr=19467):

```bash
Run yarn prettier:check
Checking formatting...
All matched files use Prettier code style!
```
Can you maybe point me out to what's missing?